Makefile

```diff
@@ -68,8 +79,9 @@ BASE_FLAGS := ${BUILD_FLAGS} -tags "${BUILDTAGS}"
 BASE_LDFLAGS := -s -w -X ${PROJECT}.gitCommit=${COMMIT} -X ${PROJECT}.version=${VERSION}
 
 # Specific build flags for build type.
-DYN_BUILD_FLAGS := ${BASE_FLAGS} -buildmode=pie -ldflags "${BASE_LDFLAGS}"
-TEST_BUILD_FLAGS := ${BASE_FLAGS} -buildmode=pie -ldflags "${BASE_LDFLAGS} -X ${PROJECT}/pkg/testutils.binaryType=test"
+DYN_BUILD_FLAGS := ${MOD_VENDOR} ${BASE_FLAGS} ${GO_BUILDMODE} -ldflags "${BASE_LDFLAGS}"
+LOCAL_VALIDATE_BUILD_FLAGS := ${BASE_FLAGS} ${GO_BUILDMODE} -ldflags "${BASE_LDFLAGS}"
```

Comment:
I don't think this is necessary -- have you checked whether removing `GO111MODULE=off` in `.travis.yml` fixes the problem? The reason we originally added it was to work around some weird build errors on older Go versions which might've been fixed by now.